CHANGELOG.md

```diff
@@ -25,7 +25,8 @@ librdkafka v2.2.0 is a feature release:
    Add DNS alias support for secured connection (#4292).
  * [KIP-339](https://cwiki.apache.org/confluence/display/KAFKA/KIP-339%3A+Create+a+new+IncrementalAlterConfigs+API):
    IncrementalAlterConfigs API (started by @PrasanthV454, #4110).
- * [KIP-554](https://cwiki.apache.org/confluence/display/KAFKA/KIP-554%3A+Add+Broker-side+SCRAM+Config+API): Add Broker-side SCRAM Config API (#4241).
+ * [KIP-554](https://cwiki.apache.org/confluence/display/KAFKA/KIP-554%3A+Add+Broker-side+SCRAM+Config+API): Add Broker-side SCRAM Config API (#4241). 
+ * Fix for idempotent producer fatal errors, triggered after a possibly persisted message state (#4438).
```

Comment:
The fix should go in a 2.2.1 maintenance release with only this item

Reply:
Is customer using Windows only? Otherwise we should upgrade Linux dependencies too.

Reply:
`RD_KAFKA_VERSION` must be changed to `020201` and version in `vcpkg.json` must be `2.2.1` too.

Reply:
Moved the changelog to v.2.2.1 and updated `RD_KAFKA_VERSION`

Reply:
In `rdkafkacpp.h` too, please